### PR TITLE
Paper under computer vision added

### PR DESCRIPTION
**Pull request by nbro**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -40,7 +36,15 @@ Markdown format:
 - [Talks](#talks)
 
 ## Computer Vision
+
 ### Image Representation Learning
+
+#### 2009
+
+- Learning Long-Range Vision for Autonomous Off-Road Driving
```
Comment (jason718) [inline README.md]: Is it better to move this to Robotic section?

**Comment by jason718** [comment]
Awesome job, only one comment. Please let me know your opinion.

File: README.md (hunk 1 above)
Comment (nbro) [reply README.md]: Yes, you're right. I will move it there.

Comment (nbro) [reply README.md]: Fixed.

**Comment by jason718** [comment]
merged! Appreciate!

**Pull request merged**